docker_services_cli/config.py

```diff
@@ -28,50 +28,100 @@
 DOCKER_SERVICES_FILEPATH = "docker_services_cli/docker-services.yml"
 """Docker services file default path."""
 
-DEFAULT_VERSIONS = {
-    "ES_6_LATEST": "6.8.12",
-    "ES_7_LATEST": "7.9.0",
-    "POSTGRESQL_9_LATEST": "9.6.19",
-    "POSTGRESQL_10_LATEST": "10.14",
-    "POSTGRESQL_11_LATEST": "11.9",
-    "POSTGRESQL_12_LATEST": "12.4",
-    "POSTGRESQL_13_LATEST": "13.0",
-    "MYSQL_5_LATEST": "5.7.31",
-    "MYSQL_8_LATEST": "8.0.21",
-    "REDIS_6_LATEST": "6.0.6",
-    "MEMCACHED_LATEST": "1.6.6",
-    "RABBITMQ_3_LATEST": "3.8.7",
-}
-"""Services default latest versions."""
-
 # Elasticsearch
 ELASTICSEARCH = {
-    "ES_VERSION": "ES_7_LATEST"
+    "ELASTICSEARCH_VERSION": "ELASTICSEARCH_7_LATEST",
+    "DEFAULT_MINOR_VERSIONS": {
+        "ELASTICSEARCH_6_LATEST": "6.8.12",
+        "ELASTICSEARCH_7_LATEST": "7.9.0",
+    },
 }
 """Elasticsearch service configuration."""
 
 # PostrgreSQL
 POSTGRESQL = {
     "POSTGRESQL_VERSION": "POSTGRESQL_9_LATEST",
-    "POSTGRESQL_USER": "invenio",
-    "POSTGRESQL_PASSWORD": "invenio",
-    "POSTGRESQL_DB": "invenio",
+    "DEFAULT_MINOR_VERSIONS": {
+        "POSTGRESQL_9_LATEST": "9.6.19",
+        "POSTGRESQL_10_LATEST": "10.14",
+        "POSTGRESQL_11_LATEST": "11.9",
+        "POSTGRESQL_12_LATEST": "12.4",
+        "POSTGRESQL_13_LATEST": "13.0",
+    },
+    "CONTAINER_CONFIG_ENVIRONMENT_VARIABLES": {
+        "POSTGRESQL_USER": "johndoe",
+        "POSTGRESQL_PASSWORD": "password",
+        "POSTGRESQL_DB": "database",
+    },
+    "CONTAINER_CONNECTION_ENVIRONMENT_VARIABLES": {
+        "SQLALCHEMY_DATABASE_URI":
+        "postgresql+psycopg2://johndoe:password@localhost:5432/database",
+    },
 }
 """Postgresql service configuration."""
 
 # MySQL
 MYSQL = {
     "MYSQL_VERSION": "MYSQL_5_LATEST",
-    "MYSQL_USER": "invenio",
-    "MYSQL_PASSWORD": "invenio",
-    "MYSQL_DB": "invenio",
-    "MYSQL_ROOT_PASSWORD": "invenio",
+    "DEFAULT_MINOR_VERSIONS": {
+        "MYSQL_5_LATEST": "5.7.31",
+        "MYSQL_8_LATEST": "8.0.21"
+    },
+    "CONTAINER_CONFIG_ENVIRONMENT_VARIABLES": {
+        "MYSQL_USER": "johndoe",
+        "MYSQL_PASSWORD": "password",
+        "MYSQL_DB": "database",
+        "MYSQL_ROOT_PASSWORD": "password",
+    },
+    "CONTAINER_CONNECTION_ENVIRONMENT_VARIABLES": {
+        "SQLALCHEMY_DATABASE_URI":
+        "mysql+pymysql://johndoe:password@localhost:3306/database",
+    },
 }
 """MySQL service configuration."""
 
-SERVICES = [
-    ELASTICSEARCH,
-    POSTGRESQL,
-    MYSQL,
-]
+REDIS = {
+    "REDIS_VERSION": "REDIS_6_LATEST",
+    "DEFAULT_MINOR_VERSIONS": {"REDIS_6_LATEST": "6.0.6"},
```

Comment:
moderate: here it is named `...MINOR...`. However, I see no `MAJOR` definition anywhere and even [below](https://github.com/inveniosoftware/docker-services-cli/pull/16/files#diff-1fc38c9800c4622657c57dc615e70a5f0ff866314547364b8338fdef5a116d86R100) major versions are taken from the `minor` lists. Should it be simple "versions"

Reply:
I am fine with changing it back, it just helped me to understand/read things better while implementing this change.

Reply:
I'll leave it up to @lnielsen, for me it was confusing and I was actually looking for the major definition.

Reply:
No need to spend time on this, I've changed it!